### ISSUE TITLE
Add support for missing instance types

### ISSUE THREAD

**Issue by efekarakus** [aws/amazon-ecs-cli#763]
### Summary
We're missing support for several  EC2 instance types.

### Description
* Which specific command was run? ` ecs-cli up ---size 1  --instance-type m5a.large --launch-type EC2 --cluster m5alarge --region us-west-2`
* Which version of the CLI you are using? v1.14.0

Output:
```
FATA[0000] Error executing 'up': ValidationError: Parameter EcsInstanceType failed to satisfy constraint: must be a valid EC2 instance type.
```

Add support for the following instance types:
- m5a.large
- m5a.xlarge
- m5a.2xlarge
- m5a.4xlarge
- m5a.12xlarge
- m5a.24xlarge
- m5ad.large
- m5ad.xlarge
- m5ad.2xlarge
- m5ad.4xlarge
- m5ad.12xlarge
- m5ad.24xlarge
- c5n.large
- c5n.xlarge
- c5n.2xlarge
- c5n.4xlarge 
 - c5n.9xlarge 
 - c5n.18xlarge
- r5a.large
- r5a.xlarge
- r5a.2xlarge
- r5a.4xlarge
- r5a.12xlarge
- r5a.24xlarge
- r5ad.large
- r5ad.xlarge
- r5ad.2xlarge
- r5ad.4xlarge
- r5ad.12xlarge
- r5ad.24xlarge
- f1.4xlarge
- f1.16xlarge
- g3s.xlarge
- p3dn.24xlarge

**Comment by efekarakus** [comment]
This issue is pending release and is fixed by https://github.com/aws/amazon-ecs-cli/commit/63db43e72b37e27560db5442fed96f6abc5e222a

**Comment by efekarakus** [comment]
This got fixed with release v1.14.1 https://github.com/aws/amazon-ecs-cli/releases/tag/v1.14.1!
